libs/designsystem/page/src/page.component.scss

```
@@ -253,7 +253,7 @@ ion-content {
   --background: #{utils.get-color('background-color')};
   --color: #{utils.get-color('black')};
 
-  @media (min-width: map.get(utils.$breakpoint-map, 'sm')) {
+  @include utils.media('>=medium') {
```

Shouldn't this be `>=632px`?:
```suggestion
  @include utils.media('>=small') {
```

---

I just double checked with @henrikvoetmand and he confirms that it should be `medium` `>=768px`